test/benchmarks/sails.load.test.js

```diff
@@ -205,10 +206,6 @@ function setupBenchmarks() {
  * @this {Array} benchmarks
  */
 function reportBenchmarks() {
-  var _ = require('lodash');
```

Comment:
@markelog Thanks for pointing that out :+1: 

re: lodash 3, see https://github.com/balderdashy/sails/issues/3582, https://github.com/balderdashy/sails/issues/3582#issuecomment-185288052, and https://github.com/balderdashy/sails/pull/3230#issuecomment-140584361